scss/components/_tabs.scss

```diff
@@ -138,7 +138,7 @@ $tab-content-padding: 1rem !default;
   display: none;
   padding: $padding;
 
-  &[aria-hidden="false"] {
+  &[aria-hidden=""] {
```

Comment:
no @DaSchTour .... its not working at all .... all div's are getting a `display: block` in that situation!